Add options (#188)
Platane/snk: packages/gif-creator/README.md

Generate a gif file from the grid and snake path.

# Implementation

It uses `gif-encoder-2` to create a gif and optimize it with `gifslice`.

**Transparency**

Having a transparent background seems to have a quite negative impact on the file size and duration. It should be avoided by default.

> Transparency is currently broken for `gif-encoder-2`

Reviewer: ```suggestion
Transparency is currently broken for `gif-encoder-2`.
```

> **Interpolaction**

Reviewer: ```suggestion
**Interpolation**
```

> It is possible to interpolate betwen two snake step for a smoother movement. This again has a huge impact on the file size and should be avoided.

Reviewer: ```suggestion
It is possible to interpolate between two snake steps for a smoother movement. This again has a huge impact on the file size and should be avoided.
```
